File: app/src/main/java/com/thoughtworks/china/mobile/varval/ValVar.java
package com.thoughtworks.china.mobile.varval;

public class ValVar {

    private String phase;
    private static final String mobileOpenDay = "Mobile Open Day";
    private String topics = "《三生三世iOS布局》\n"
            + "《移动测试的Mock实践》\n"
            + "《Getting Start with Kotlin on Android》";

    public static void main(String[] args) {
        ValVar valVar = new ValVar();
        valVar.phase = "4th";
        System.out.println("Welcome to "
                + mobileOpenDay
                + " "
                + valVar.phase);
        System.out.println(String.format("Welcome to %s %s",
                mobileOpenDay, valVar.phase));
        System.out.println(valVar.topics);
    }
}

File: app/src/main/kotlin/com/thoughtworks/china/mobile/varval/ValVar.kt
package com.thoughtworks.china.mobile.varval

object ValVarKT {

    private var phase: String? = null
    private const val mobileOpenDay = "Mobile Open Day"
    private val topics = """
    |《三生三世iOS布局》
    |《移动测试的Mock实践》
    |《Getting Start with Kotlin on Android》
    """.trimMargin()

    @JvmStatic
    fun main(args: Array<String>) {
        val valvar = ValVarKT
        valvar.phase = "4th"

        println("Welcome to " + mobileOpenDay + " " + valvar.phase)
        println("Welcome to $mobileOpenDay ${valvar.phase}")
        println(topics)
    }
}